doc/how_to/custom_components/reactive_html/reactive_html_layout.md

```diff
@@ -0,0 +1,292 @@
+# Create Layouts With ReactiveHTML
+
+In this guide you will learn how to build custom layouts using HTML and `ReactiveHTML`.
+
+If you are not familiar with HTML then the [W3 HTML School](https://www.w3schools.com/html/default.asp)
+is a good resource to learn from. You can also ask ChatGPT for help. It can often provide you with
+a good HTML starting point that you can fine tune.
+
+## A layout of a single object
+
+You can layout a single object as follows.
+
+```{pyodide}
+import panel as pn
+import param
+
+pn.extension()
+
+class LayoutSingleObject(pn.reactive.ReactiveHTML):
+    object = param.Parameter()
+
+    _ignored_refs = ("object",)
```

Comment:
If I don't add `object` to `_ignored_refs` its shown as a literal value. Is that a bug?